apps/studio/pages/project/[ref]/auth/smtp.tsx

```diff
@@ -0,0 +1,75 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+
+import { SmtpForm } from 'components/interfaces/Auth'
+import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import { useParams } from 'common'
+import { useCurrentPath } from 'hooks/misc/useCurrentPath'
+import Link from 'next/link'
+import { NavMenu, NavMenuItem } from 'ui'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldDivider,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
+import type { NextPageWithLayout } from 'types'
+
+const PageLayout: NextPageWithLayout = () => {
+  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+  const isPermissionsLoaded = usePermissionsLoaded()
+  const { ref } = useParams()
+  const currentPath = useCurrentPath()
+
+  const navMenuItems = [
+    {
+      label: 'Templates',
+      href: `/project/${ref}/auth/templates`,
+    },
+    {
+      label: 'SMTP Settings',
+      href: `/project/${ref}/auth/smtp`,
+    },
+  ]
+
+  if (isPermissionsLoaded && !canReadAuthSettings) {
+    return <NoPermission isFullPage resourceText="access your project's email settings" />
+  }
+
+  return (
+    <div>
+      <ScaffoldHeader className="pb-0">
+        <ScaffoldContainer id="auth-page-top">
+          <ScaffoldTitle>Email Settings</ScaffoldTitle>
```

Comment:
Wondering if we should match the title of the page to the side nav label?

![image](https://github.com/user-attachments/assets/c8ff335e-8bf5-40c5-bef7-18685438efdc)
![image](https://github.com/user-attachments/assets/7e30b9b3-8b4b-4899-b534-3d974e822f02)